RELEASE-NOTES.md

```diff
@@ -21,6 +21,11 @@
 
 ## New additions
 * Added support for `title` field in Streamlit definition in `snowflake.yml` project file.
+* Added support for `--env` command line arguments for templating.
+  * Available for commands that make use of the project definition file.
+  * Format of the argument: `--env key1=value1 --env key2=value2`.
+  * Overrides `env` variables values when used in templating.
```

Comment:
Does it override system-defined environment variables? I'm guessing yes

Reply:
no, it does not alter them.

Reply:
Can you explain the hierarchy for the inputs?

Reply:
Sure i can add that.